### ISSUE TITLE
Problem download

### ISSUE THREAD

**Issue by 22marco77** [Scotty-Retro/Neon-BlastV1#12]
I can't download your theme which I think is awesome, every time I download the theme when I try to extract it it says file corrupted, would you be so kind to check that your file is all ok??? thanks

**Comment by Noruas29** [comment]
I can confirm, I have exactly the same problem.

**Comment by 22marco77** [comment]
ok

**Comment by 22marco77** [comment]
the creator of the theme will no longer be interested in solving the problem because he hasn't solved the download bug and I always get the same error, evidently this resource has been lost, too bad because the theme from my point of view is the best and most beautiful .

**Comment by ISeeTWizard** [comment]
For me it's a GitHub Problem as the theme is very very large...
Downloading it manually and copying it to my MicroSD worked well for me...

**Comment by Noruas29** [comment]
I tried but there are lost files and I had errors.... It seems to be the origin of the problem.

**Comment by 22marco77** [comment]
That's the problem and Scott has no interest in solving it... Such a shame

**Comment by OldGamers** [comment]
Hi Everyone I took the time and made backup for everyone to use the theme. "Im not creator of the theme" just happen I had it when first came out. Please Understand you Need very HIGH Speed Ethernet to download the Master file in zip format. at lease have 100mb of speed or higher. Having low net speed or wifi will not work in order to download this theme. I understand the file so large that no one cant get pass the 6gb mark for this get threw the retrobat download or from here. Here a Google Drive download for those want to use it for the Retrobat.
[https://drive.google.com/file/d/1yYuFQz_TB6L5cO6EW7Ml_SDR9HsOWOpI/view?usp=sharing](url)

How to Install and Use the Neon Blast V1 on the RetroBat Frontend
1. extract the Neon Blast V1-masters 
2. take the Neon Blast V1-masters folder and find your retrobat folder 
3. this is where is need to be located RetroBat Folder > emulationstation folder> .emulationstation folder > Themes folder > Neon-BlastV1-master

**Comment by 22marco77** [comment]
Thanks for replying, but when I open the link you indicated it takes me back to Scott's page, I am attaching the screenshot
![Senza titolo](https://github.com/Scotty-Retro/Neon-BlastV1/assets/24650739/35631bed-2400-4208-8a38-d24a1987b143)

**Comment by 22marco77** [comment]
Using another browser I was able to access the address indicated, thanks ... now I'm trying to download the file hoping it will be successful. THANKS AGAIN

**Comment by Scotty-Retro** [comment]
Hello guys. Sorry for the late reply. The only problem is that the theme is very large and still growing. From what I understand is that it has been failing in certain browsers personally I use chrome and have never had a problem downloading it from github So I'm not sure what can be fixed from my end

**Comment by 22marco77** [comment]
> Hello guys. Sorry for the late reply. The only problem is that the theme is very large and still growing. From what I understand is that it has been failing in certain browsers personally I use chrome and have never had a problem downloading it from github So I'm not sure what can be fixed from my end 

Thanks for the reply even if late it doesn't matter, better late than never. First of all I would like to congratulate you for the theme, the best for me, it's really great, the only thing I ask you how I can integrate the whell of some systems that are missing, like mame, the various Atari etc ... I notice that using the wheel clubs many are absent and the color does not allow you to read the name of the system well

**Comment by Scotty-Retro** [comment]
> 
are you using Batocera? if so I think you just need to change the logos names to lower case or the correct names all Atari systems are covered and should all have logos

**Comment by 22marco77** [comment]
Yes I use Batocera, isn't your theme native to Batocera? Would it be possible to adapt it also for retropies

**Comment by 22marco77** [comment]
I batocera on Raspberry pi 4 8gb ram, I noticed that using your theme often in the system change or after a break the rp pi4 freeza, using the carbon base theme this does not happen... How come

**Comment by Scotty-Retro** [comment]
> I batocera on Raspberry pi 4 8gb ram, I noticed that using your theme often in the system change or after a break the rp pi4 freeza, using the carbon base theme this does not happen... How come

I think because there's alot going on with neon blast the pi4 probably struggling to handle it or your pi maybe overheating if it is overclocked could be a few reasons you could try disabling animations see if that helps any

**Comment by 22marco77** [comment]
> > I batocera on Raspberry pi 4 8gb ram, I noticed that using your theme often in the system change or after a break the rp pi4 freeza, using the carbon base theme this does not happen... How come
> 
> I think because there's alot going on with neon blast the pi4 probably struggling to handle it or your pi maybe overheating if it is overclocked could be a few reasons you could try disabling animations see if that helps any
> 
Ok thanks for the information, disable animations and down overclocking to a minimum. I noticed that the club type wheels of the systems are missing that of mame and Atari, I only read the writing but I do not have the carbon type plate to make me understand. How can I insert it

**Comment by Scotty-Retro** [comment]
> > > I batocera on Raspberry pi 4 8gb ram, I noticed that using your theme often in the system change or after a break the rp pi4 freeza, using the carbon base theme this does not happen... How come
> > 
> > 
> > I think because there's alot going on with neon blast the pi4 probably struggling to handle it or your pi maybe overheating if it is overclocked could be a few reasons you could try disabling animations see if that helps any
> 
> Ok thanks for the information, disable animations and down overclocking to a minimum. I noticed that the club type wheels of the systems are missing that of mame and Atari, I only read the writing but I do not have the carbon type plate to make me understand. How can I insert it

https://drive.google.com/file/d/1R-A8Z8CJ2jb61AuOO9_IKeP8noXXaPk4/view?usp=sharing

Mame and atari logos just check they're named correctly for batocera and place in the correct folders but also remember batocera is case sensitive

**Comment by 22marco77** [comment]
The names of the rom systems created at the time of installation of batocera have not been changed, I only inserted the rom in the folders as indicated by the log file inserted in it. Not renamed any folder I simply used the structure already created by the system. And when I turn the wheel with the names of the systems some of these do not have the neon blast wheel but I only see an inscription of the name of the system colored black difficult to read. I can send you a photo to make me understand better
